File: Volume_Funcional.py

def volume_esfera(pi, r):
        volume = 4 * pi * r * r  * r  * r / 3
        print(volume)

def volume_piramide(area_base, h):
        volume = area_base * h / 3
        print(volume)

def volume_cilindro(pi, r, h):
        volume = pi * r * r  * h
        print(volume)
        
e = volume_esfera(3.14, 10)
p = volume_piramide(10, 15)
c = volume_cilindro(3.14, 15, 10)

print(e)
print(p)
print(c)

File: Volume_Funcional.c
#include <stdio.h>
    void volume_esfera(float r, float pi, float h) {
        float volume = 4 * pi * r * r * r/ 3;
        printf("%f \n", volume);
    }

    void volume_piramide(float area_base, float h) {
        float volume = area_base * h / 3;
        printf("%f \n", volume);
    }
    void volume_cilindro(float r, float pi, float h) {
        float volume = pi * r * r * h;
        printf("%f \n", volume);
    }

int main() {

    float pi = 3.14;
    float r = 10;
    float h = 15;
    float area_base = 20;

   
   volume_esfera(r, pi, h);
   volume_piramide(area_base, h);
   volume_cilindro(r, pi, h);




    return 0;
}
